frontend/src/components/Select.jsx

```diff
@@ -0,0 +1,17 @@
+export default function Select({label, name, items, value, onChange}) {
+  return (
+    <div>
+      <label for="select-input-default" class="rvt-label" onChange={onChange}>{label}</label>
+      <select id="select-input-default" class="rvt-select">
+        {
+          items.map(item => (
+            <option value="option one">{option.label}</option>
+          ))
+        }
+        <option value="option one">Option One</option>
```

Comment:
Are these default values? I don't see them show up on the web app so just curious, doesn't seem like there'd be any effect with removing these (but would advise doing so for anyone developing this in the future to have better clarity)